### DISCUSSION TITLE
Backdrop element overlapping with emoji picker

### DISCUSSION THREAD

**Discussion by rimzzlabs** [contentco/quill-emoji#167] [Q&A]
Great extension for quill thank you, Im currently using it with my React app, and here's what my problem is:

This is the backdrop element, appear on the root of the body tag, while my emoji element is inside my `<ReactQuill />` element
![image](https://user-images.githubusercontent.com/62492410/182763697-5b73d31e-06a4-42d7-9c2f-6344abe3e77f.png)

I couldn't pick any emoji because the backdrop element z-index was too big with the emoji picker element, is there any solution?

**Comment by rahXephonz** [comment]
Are you was trying custom the style?
